recipes/libgpiod/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${CMAKE_PROJECT_NAME} test_package.cpp)
```

Comment:
Please test the c library (`libgpiod`) and convert `test_package.cpp` to a pure c source.

Eventually, you can also test the cxx bindings by optionally building a `test_package.cpp` file.